content/en/blog/2021/migrate-alpha-security-policy/index.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Migrate the deprecated alpha security policy
+description: A tutorial to help customers to migrate the deprecated alpha security policy to the beta version.
+publishdate: 2021-02-01
+attribution: Yangmin Zhu (Google)
+keywords: [security,policy,migrate,alpha,beta,deprecate,peer,jwt,authorization]
+---
+
+Istio beta security policy (`PeerAuthentication`, `RequestAuthentication` and `AuthorizationPolicy`) has been released long
+time ago and the alpha security policy (`MeshPolicy`, `Policy`, `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`) has been
+deprecated and no longer supported starting 1.6.
+
+It is required to migrate the alpha security policy to the beta version before upgrading to Istio 1.6 and later.
+This blog is a tutorial to help customers who are still using the alpha security policy in old Istio to migrate to beta
+security policy and unblock the upgrade to Istio 1.6 and later.
+
+## Major changes in `v1beta1` security policy
+
+Please note there are non-backward compatible changes in the `v1beta1` policy. The major changes related to the migration
+are listed below:
+
+| Feature | `v1alpha1` Policy | `v1beta1` Policy |
+|---------|------------------------|--------------------------------|
+| API stability | **No** backward compatible | backward compatible **guaranteed** |
+| mTLS related CRDs | `MeshPolicy` and `Policy` | `PeerAuthentication` |
+| JWT related CRDs | `MeshPolicy` and `Policy` | `RequestAuthentication` and `AuthorizationPolicy` |
```

Comment:
Features can just be "mTLS authentication" and "JWT authentication". You don't need "related CRDs".

Reply:
And let's separate Authorization and JWT authentication.